src/App.tsx

```diff
@@ -1,27 +1,65 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+  const getAllGoods = async () => {
+    const data = await getAll();
 
-    <button type="button" data-cy="first-five-button">
-      Load 5 first goods
-    </button>
+    setGoods(data);
+  };
 
-    <button type="button" data-cy="red-button">
-      Load red goods
-    </button>
+  const getFirst5Goods = async () => {
+    const data = await get5First();
 
-    <GoodsList goods={[]} />
-  </div>
-);
+    setGoods(data);
+  };
+
+  const getRedColorGoods = async () => {
+    const data = await getRedGoods();
+
+    setGoods(data);
+  };
```

Comment:
Let's create a `loadGoods` function and pass then a callback (getAll, get5First, getRedGoods), instead of creating 3 similar functions

Reply:
```
 loadGoods = async (callback: () => Promise<Good[]>) => {
		....
		}
```